docs/proposals/working-groups/20221115-managed-kubernetes.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Working Group for Managed Kubernetes in Cluster API
+authors:
+  - "@jackfrancis"
+reviewers:
+    - "@pydctw”"
+    - "@richardcase"
+    - "@vincepri"
+    - "@sbueringer”"
+    - "@fabriziopandini"
+creation-date: 2022-11-15
+last-updated: 2022-11-15
+status: proposed
+see-also:
+  - https://github.com/kubernetes-sigs/cluster-api/issues/7494
+  - https://github.com/kubernetes-sigs/cluster-api/issues/6988
+---
+# Managed Kubernetes in Cluster API Working Group
+
+This document briefly outlines the scope, communication media, and stakeholders for a formal Working Group dedicated to defining a Cluster API-approved Managed Kubernetes solution.
+
+## Scope
+
+The charter for this Working Group will be standardizing Managed Kubernetes solutions across the Cluster API provider ecosystem.
+
+## Communication
+
+We will meet on [Wednesdays at 09:00 PT (Pacific Time)](https://zoom.us/j/861487554). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=09:00&tz=PT%20%28Pacific%20Time%29).
```

Comment:
@pydctw @richardcase should we do a doodle or equivalent to crowd source a regular meeting time and frequency?

Reply:
Sounds good to me 👍

Reply:
Wed 9am Pacific sounds good to me but we can do a doodle for broader audience 👍

Reply:
That would work for me as well. We could start with that time and then change in the future if needed.

Reply:
With 9 PT being quite late for IST will we possibly record the meetings?

Reply:
Thanks everyone for the comments! To be clear we won't do this today, I'll put this as an agenda item for office hours and we can aim to 1) doodle + 2) host our first session next week.

@joekr I'm in favor of recording, I'll add that to the PR and other folks can chime in if there is not consensus there

Reply:
Here's a doodle with a few choices for time slot:

https://doodle.com/meeting/organize/id/aK1Gnjnd

We can probably choose in this thread whether to do weekly or bi-weekly or monthly. (My vote is weekly to start.)

cc @oscr @pmoust

Reply:
Agreed on weekly to start.

Reply:
Updated doc w/ the consensus-determined time of 9 a.m. PST, thanks all!